### PR TITLE
LPS-195090 AutoComplete element dropdown does not have the same width from the respective input field

### PR DESCRIPTION
**Pull request by Artartur**
Related Issue: [LPS-195090](https://liferay.atlassian.net/browse/LPS-195090)

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutField.tsx [hunk 1]
```diff
@@ -179,6 +179,7 @@ export default function ModalAddObjectLayoutField({
 							'there-are-no-fields-for-this-object'
 						)}
 						error={errors.objectFieldName}
+						id="modal-add-object-layout-field"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutField.tsx:182]: Hi @Artartur. Please, rename the id to `modalAddObjectLayoutField`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx [hunk 2]
```diff
@@ -261,6 +261,7 @@ export function ModalAddObjectLayoutTab({
 								'there-are-no-relationship-for-this-object'
 							)}
 							error={errors.objectRelationshipId}
+							id="modal-add-object-layout-tab"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx:264]: Hi @Artartur. Please, rename the id to `modalAddObjectLayoutTab`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx [hunk 3]
```diff
@@ -464,6 +464,7 @@ export function ModalAddFilter({
 							'there-are-no-columns-available'
 						)}
 						error={errors.selectedFilterBy}
+						id="modal-add-filter-by"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx:467]: Hi @Artartur. Please, rename the id to `modalAddFilterBy`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ScopeContainer.tsx [hunk 4]
```diff
@@ -134,6 +134,7 @@ export function ScopeContainer({
 					'no-options-were-found'
 				)}
 				error={errors.titleObjectFieldId}
+				id="scope-container"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ScopeContainer.tsx:137]: Hi @Artartur. Please, rename the id to `objectDetailsScopeContainer`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx [hunk 5]
```diff
@@ -345,6 +345,7 @@ export function AggregationFormBase({
 					'no-relationships-were-found'
 				)}
 				error={errors.objectRelationshipName}
+				id="aggregation-relationship"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx:348]: Hi @Artartur. Please, rename the id to `objectFieldAggregationFormBase`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx [hunk 6]
```diff
@@ -391,6 +392,7 @@ export function AggregationFormBase({
 						'no-fields-were-found'
 					)}
 					error={errors.objectFieldName}
+					id="aggregation-field"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx:395]: Hi @Artartur. Please, rename the id to `objectFieldAggregationField`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx [hunk 7]
```diff
@@ -441,6 +441,7 @@ export default function ObjectFieldFormBase({
 					disabled={disabled}
 					emptyStateMessage={Liferay.Language.get('option-not-found')}
 					error={errors.listTypeDefinitionId}
+					id="object-field-form-base"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx:444]: Hi @Artartur. Please, rename the id to `objectFieldFormBase`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectDefinition.tsx [hunk 8]
```diff
@@ -68,6 +68,7 @@ export default function SelectObjectDefinition({
 			disabled={disabled}
 			emptyStateMessage={Liferay.Language.get('no-objects-were-found')}
 			error={error}
+			id="select-object-definition"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectDefinition.tsx:71]: Hi @Artartur. Please, rename the id to `objectRelationshipSelectObjectDefinition`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectRelationship.tsx [hunk 9]
```diff
@@ -90,6 +90,7 @@ export default function SelectRelationship({
 		<AutoComplete<LabelNameObject>
 			emptyStateMessage={Liferay.Language.get('no-parameters-were-found')}
 			error={error}
+			id="select-object-relationship"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectRelationship.tsx:93]: Hi @Artartur. Please, rename the id to `objectRelationshipSelectObjectRelationship`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/BasicInfo.tsx [hunk 10]
```diff
@@ -162,6 +162,7 @@ export function BasicInfo({
 									'no-fields-were-found'
 								)}
 								error={errors.outputType}
+								id="basic-info"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/BasicInfo.tsx:165]: Hi @Artartur. Please, rename the id to `objectValidationBasicInfo`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Conditions.tsx [hunk 11]
```diff
@@ -189,6 +189,7 @@ export function Conditions({
 								'no-fields-were-found'
 							)}
 							error={errors.outputType}
+							id="conditions"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Conditions.tsx:192]: Hi @Artartur. Please, rename the id to `objectValidationConditions`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx [hunk 12]
```diff
@@ -135,6 +135,7 @@ export function ModalAddDefaultSortColumn({
 							emptyStateMessage={Liferay.Language.get(
 								'there-are-no-columns-added-in-this-view-yet'
 							)}
+							id="modal-add-default-sort-column"
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx:138]: Hi @Artartur. Please, rename the id to `objectViewModalAddDefaultSortColumn`.

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @Artartur. I just asked some renames and also changing the format to camel case. Thanks!

**Comment by Artartur** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutField.tsx (hunk 1 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutField.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx (hunk 2 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx (hunk 3 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ScopeContainer.tsx (hunk 4 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ScopeContainer.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx (hunk 6 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx (hunk 7 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectDefinition.tsx (hunk 8 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectDefinition.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectRelationship.tsx (hunk 9 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectRelationship.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Conditions.tsx (hunk 11 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Conditions.tsx]: Done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx (hunk 12 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx]: Done!

**Comment by alinedoleron** [comment]
ci:test:object

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:sf

**Comment by Artartur** [comment]
ci:test: relevant

**Comment by Artartur** [comment]
ci:test:object

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
Hi @Artartur. Please, could you rebase the code to see if solve those failures? Thanks!

**Comment by alinedoleron** [comment]
ci:forward:force

**Comment by davisaints** [comment]
ci:test:object

**Comment by davisaints** [comment]
ci:test:relevant

**Comment by davisaints** [comment]
ci:test:stable

**Comment by davisaints** [comment]
This pull request is good to go!

**Comment by davisaints** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward:force

**Pull request closed without merging**